detections/application/splunk_digital_certificates_infrastructure_version.yml

```diff
@@ -0,0 +1,51 @@
+name: Splunk Digital Certificates Infrastructure Version
+id: 3c162281-7edb-4ebc-b9a4-5087aaf28fa7
+version: 1
+date: '2022-05-26'
+author: Lou Stella, Splunk
+type: Hunting
+datamodel: []
+description: On June 14th, 2022, Splunk released a security advisory relating to TLS validation occuring between different parts of Splunk Enterprise Infrastructure. In addition to upgrading to Splunk Enterprise 9.0 or later, several configuration settings need to be set. This search will check those configurations on the search head it is run from as well as its search peers. Other components such as additional search heads or anything this rest command cannot be distributed to will need to be manually checked. 
+search: '| rest /services/server/info | table splunk_server version server_roles | join splunk_server [| rest /servicesNS/nobody/search/configs/conf-server/ search="sslConfig"| table splunk_server sslVerifyServerCert sslVerifyServerName serverCert] | fillnull value="Not Set" | rename sslVerifyServerCert as "Server.conf:SslConfig:sslVerifyServerCert", sslVerifyServerName as "Server.conf:SslConfig:sslVerifyServerName", serverCert as "Server.conf:SslConfig:serverCert" | `splunk_digital_certificates_infrastructure_version_filter`'
```

Comment:
Do these rest calls need to be dispatched to the indexers? Users might not have that permission. Can remove that error message if we restrict it to searchhead with.... splunk_server=local count=0.